### ISSUE TITLE
:bulkrename swapping names

### ISSUE THREAD

**Issue by onethreethree7** [ranger/ranger#1621]
#### Current Behavior
Using bulkrename if any two names of files are swapped, then one of the files will be deleted by mv, as in ```mv file1 file2; mv file2 file1``` will be executed.
#### Expected Behavior
It would be nice if it somehow handled this, by perhaps using temporary files.

**Comment by toonn** [comment]
Yep, our `:bulkrename` should come with a HUGE warning. Alternatives like `vidir` and, I think, `dired` should be much safer.
@Vifon, did some work to replace `:bulkrename` with `vidir`. But not everyone's in agreement we should require a dependency for this.

Maybe in the meanwhile we should at least add a warning banner to the tempfile or something?
